recipes/postgresql/bld.bat

```diff
@@ -0,0 +1,28 @@
+cd src\tools\msvc
```

Comment:
Since Windows is skipped, I'd be inclined to just remove this file? Would you prefer to keep it?